Running-Mastodon/PgBouncer-guide.md

```diff
@@ -87,7 +78,7 @@ listen_addr = 127.0.0.1
 listen_port = 6432
 ```
 
-Put `md5` as the `auth_type` (assuming you're using the md5 format in `userlist.txt`):
+Put `md5` as the `auth_type`:
```

Comment:
Found the problem with the md5. 

It is `('md5'+md5(password+user))` and not `('md5'+md5(user+password))` [source](https://github.com/pgbouncer/pgbouncer/pull/120/files)

Did the changes and added the info that auth_type md5 allows both md5 and plain-text